File: YLF/YLF.java
import java.util.Scanner;

public class Program
{
    public static void main(String[] args) {
 Scanner input = new Scanner(System.in);
 String link = input.nextLine();
 int x=link.indexOf("e/");
 int xx=link.indexOf("v=");
 if(x>0)
 System.out.print(link.substring(x+2));
 if(xx>0)
 System.out.print(link.substring(xx+2));
 }
}

File: YLF/YLF.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


namespace Sololearn
{
    class Program
    {
        static void Main(string[] args)
        {
   string link = Console.ReadLine();
   int x = link.IndexOf("e/");
   int xx = link.IndexOf("v=");
   if(x>0) 
   Console.Write(link.Substring(x+2));
   if(xx>0)
   Console.Write(link.Substring(xx+2));
         
        }
    }
}
